digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/NodeDAO.scala

```diff
@@ -337,4 +337,27 @@ class NodeDAO extends BaseDAO {
       queryList(query)
     }
   }
+
+  def fetchNodeNumbersByProject(projectId: Long): Seq[Long] = {
```

Comment:
Should we join here also the node -table and check that the node.end_date is null and node.valid_to is null.